app/models/miq_request_workflow.rb

```diff
@@ -1236,13 +1242,19 @@ def set_ws_field_value(values, key, data, dialog_name, dlg_fields)
     values[key] = set_value
   end
 
+  def apply_result(result, data_type)
```

Comment:
Can this be moved down into the `private` method section at the end of the file.